### DISCUSSION TITLE
Release v1.9.8 / 5.64.8

### DISCUSSION THREAD

**Discussion by DavidXanatos** [sandboxie-plus/Sandboxie#3051] [Announcements]
![0 5 5](https://user-images.githubusercontent.com/3890945/105014988-575cb680-5a41-11eb-90d5-d942d28184b2.png)

This is a maintenance release, it fixes a couple of issues.

Please review the full [Changelog](https://github.com/sandboxie-plus/Sandboxie/blob/master/CHANGELOG.md#198--5648---2023-06-21) for more details.

#### You can support the project through donations, any help will be greatly appreciated.

[<img src="https://www.paypalobjects.com/en_US/AT/i/btn/btn_donateCC_LG.gif">](https://xanasoft.com/go.php?to=donate)

#### If you have issues with an update installation, just uninstall the previous version keeping the sandboxie.ini and reinstall the new build.

<hr /><em>This discussion was created from the release <a href='https://github.com/sandboxie-plus/Sandboxie/releases/tag/v1.9.8'>Release v1.9.8 / 5.64.8</a>.</em>

**Comment by BilleBarrett** [comment]
I discover there are actually 2 registry locations that need clearing
HKEY_CURRENT_USER\Software\Microsoft\Windows\Shell\Associations\UrlAssociations\http\UserChoice 
HKEY_CURRENT_USER\Software\Microsoft\Windows\Shell\Associations\UrlAssociations\https\UserChoice

**Comment by BilleBarrett** [comment]
Microsoft just release 2023-06 Cumulative Update Preview for Windows 11 Version 22H2 for x64-based Systems (KB5027303); Updates; KB5027303; 6/27/2023; 127.53 GB; Pending (!)
